src/api/client.ts

```diff
@@ -7,39 +7,45 @@ import {
   searchCollectionsFieldPolicy,
 } from './field-policies';
 
-const apolloOptions = {
-  cache: new InMemoryCache({
-    typePolicies: {
-      CollectionAuthor: {
-        keyFields: ['externalId'],
-      },
-      CollectionStory: {
-        keyFields: ['externalId'],
-      },
-      Collection: {
-        keyFields: ['externalId'],
-      },
-      ApprovedCuratedCorpusItem: {
-        keyFields: ['externalId'],
-      },
-      RejectedCuratedCorpusItem: {
-        keyFields: ['externalId'],
-      },
-      ScheduledCuratedCorpusItem: {
-        keyFields: ['externalId'],
-      },
-      ScheduledCuratedCorpusItemsResult: {
-        keyFields: ['scheduledDate'],
-      },
-      Query: {
-        fields: {
-          getCollectionAuthors: getCollectionAuthorsFieldPolicy,
-          getCollectionPartners: getCollectionPartnersFieldPolicy,
-          searchCollections: searchCollectionsFieldPolicy,
-        },
+/**
+ * Custom type policies for Apollo Cache. Now in their own
+ * variable to feed them into integration tests, too.
+ */
+export const apolloCache = new InMemoryCache({
```

Comment:
When I was trying to get the last integration test going for this PR, this was identified as one of the possible issues with our integration tests - the fact we haven't previously used the Apollo cache configuration for Mocked Provider.